Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -919,6 +919,60 @@ def flipMultilinear (f : G →L[𝕜] ContinuousMultilinearMap 𝕜 E G') :
       dsimp only [MultilinearMap.coe_mk]
       exact LinearMap.mkContinuous_norm_le _ (by positivity) _
 
+/-- Flip arguments in `f : ContinuousMultilinearMap 𝕜 E (G →L[𝕜] G')` to get
+`G →L[𝕜] ContinuousMultilinearMap 𝕜 E G'` -/
+@[simps! apply_apply]
+def _root_.ContinuousMultilinearMap.flipLinear (f : ContinuousMultilinearMap 𝕜 E (G →L[𝕜] G')) :
+    G →L[𝕜] ContinuousMultilinearMap 𝕜 E G' :=
```

Comment:
Does `MultilinearMap.mkContinuousLinear` help with some boilerplate here?

Reply:
Yes, it does, thanks! Unfortunately, this means I had to shuffle a little bit the file, moving everything about `MultilinearMap.mkContinuousLinear` earlier in the file, so the diff becomes a little bit bigger.